examples/ansible/kubeone/roles/kubeone_config/defaults/main.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kubeone_versions_kubernetes: 1.14.1
+kubeone_clusterNetwork_podSubnet: 10.244.0.0/16
```

Comment:
Do we need all those variables considering this is just an example? I'm not sure how are we going to distribute those, but if it works like Terraform, I believe just a few needed and common could be enough.

Reply:
The template would be a whole lot more complex if I would check for the presence of each variable so my approach was to just take the KubeOne defaults and integrate them here